examples/Integration Mappings/Integration-mappings.ps1

```diff
@@ -0,0 +1,265 @@
+        if (!$this.OriginalAnalyticsClientName)
+        {
+            $this.OriginalAnalyticsClientName = "cli"
+        }
+
+        if ($SettingsContent.EnableanalyticsClient)
+        {
+            $this.IsOriginalAnalyticsClientEnabled = $true
+        }
+        else
+        {
+            $this.IsOriginalAnalyticsClientEnabled = $false
+        }
+
+        Write-Log "Stored original analytics client settings, name: $( $this.OriginalAnalyticsClientName ), enabled: $( $this.IsOriginalAnalyticsClientEnabled )." -output false
+    }
+
+    [void]
+    RestoreOriginalSettings()
+    {
+        if ($this.IsOriginalAnalyticsClientEnabled)
+        {
+            $RestoreAnalyticsClientEnablementState = "--enable-analytics-client"
+        }
+        else
+        {
+            $RestoreAnalyticsClientEnablementState = "--no-enable-analytics-client"
+        }
+
+        "$( box configure:settings $RestoreAnalyticsClientEnablementState --analytics-client-name=$($this.OriginalAnalyticsClientName) )"
+        Write-Log "Restored original analytics client settings, name: $( $this.OriginalAnalyticsClientName ), enabled: $( $this.IsOriginalAnalyticsClientEnabled )." -output false
+    }
+
+    [void]
+    SetScriptAnalyticsClient()
+    {
+        if ( $this.IsAnalyticsClientSupported())
+        {
+            $this.StoreOriginalSettings()
+
+            "$( box configure:settings --enable-analytics-client --analytics-client-name=$($this.TemporaryAnalyticsClientName) )"
+            Write-Log "Set temporarily analytics client settings, name: $( $this.TemporaryAnalyticsClientName ), enabled: true." -output false
+        }
+    }
+
+    [void]
+    UnsetScriptAnalyticsClients()
+    {
+        if ( $this.IsAnalyticsClientSupported())
+        {
+            $this.RestoreOriginalSettings()
+        }
+    }
+}
+
+#############################################################################
+
+# Check if any of the command-line arguments are missing
+# If so, prompt the user to enter them
+if (-not $action) {
+     Write-Log "Enter EXTRACT to grabbing current mappings or APPLY to apply new mappings:" -output true -color Yellow
+     $action = Read-Host
+}
+
+Function Start-Integrations-Mapping {
+
+    Write-Log "Starting Process" -output true -color Green
+
+    try {
+        #If a user inputs EXTRACT it will grab the current mappings and save them to a csv file
+        If ($action -eq "EXTRACT") {
+            Write-Log "Extracting current mappings" -output true -color Green
+            if (Test-Path mappings.csv) {
+                Write-Log "Removing old mappings2.csv" -output true -color Green
+                Remove-Item -Path mappings.csv -Force
+                Write-Log "File mappings2.csv removed" -output true -color Green
+            }
+            Write-Log "Pulling current mappings" -output true -color Green
+            $ExtractionResp = (box integration-mappings:slack:list --csv --save-to-file-path=mappings.csv 2>&1)
```

Comment:
to limit the payload of `box integration-mappings:slack:list` response we can even filter the fields required to our needs by adding a `fields` flag:

`--fields='id,box_item.id,partner_item.id'`